composer.json

```diff
@@ -16,17 +16,20 @@
         "forum": "https://discourse.laminas.dev"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "~8.0.0 || ~8.1.0",
         "laminas/laminas-stdlib": "^3.2.1"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "~1.0.0",
-        "laminas/laminas-json": "^3.0.0",
-        "laminas/laminas-session": "^2.8.5",
-        "phpunit/phpunit": "^9.3"
+        "laminas/laminas-coding-standard": "~2.4.0",
```

Comment:
Ah, this won't fly as-is 😬

Reply:
Shall I drop the latest commit with the coding standard update?

Reply:
Either that, or we actually upgrade the codebase to the new CS ruleset (with the exclusion of strict types, depending on how risky that part is)

Reply:
Okay, I've dropped that commit for now.

Reply:
We should still aim for a green build, before tagging, btw.